Cargo.toml

```diff
@@ -382,6 +382,7 @@ all-arch = ["wasmtime/all-arch"]
 winch = ["wasmtime/winch"]
 wmemcheck = ["wasmtime/wmemcheck"]
 memory-protection-keys = ["wasmtime-cli-flags/memory-protection-keys"]
+wasi-runtime-config = ["dep:wasmtime-wasi-runtime-config"]
```

Comment:
Could this be added to the on-by-default features list? That's generally what we're doing with WASI proposals right now so long as there's no extra build requirements, which this proposal doesn't have.